test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -617,7 +617,7 @@ public void TestTreeEnsembleCombiner()
         public void TestTreeEnsembleCombinerWithCategoricalSplits()
         {
             var dataPath = GetDataPath("adult.tiny.with-schema.txt");
-            var dataView = TextLoader.Create(Env, new TextLoader.Arguments(), new MultiFileSource(dataPath));
+            var dataView = TextLoader.ReadFile(Env, new MultiFileSource(dataPath), new TextLoader.Arguments());
```

Comment:
>ReadFile [](start = 38, length = 8)

ReadTextFile

Reply:
across whole file

---
In reply to: [239164058](https://github.com/dotnet/machinelearning/pull/1784#discussion_r239164058) [](ancestors = 239164058)

Reply:
Isn't text implied by the name of `TextLoader`?